coa-helm/values.yaml

```diff
@@ -8,9 +8,11 @@
   batchAccountKey: RUNTIME_PARAMETER
   batchNodesSubnetId: RUNTIME_PARAMETER
   drsUrl: RUNTIME_PARAMETER
+  sourceWorkspaceId: RUNTIME_PARAMETER

```

Comment:
I put this under `config` pretty much randomly; is there a better place for it?  Does it matter?

Reply:
I don't feel strongly about this either way; I think config is fine. I might suggest an entirely new section "provenance" to handle any future values of "where does this data come from?" but it might end up being a new section with only one value.

Reply:
would it make sense to make a clone section in case in the future there are more values that we need to pass (like the short term token?)

Reply:
would clone fall under provenance?  so that if i create a `provenance` section everything for cloning can go under it as well?